<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分析</string>
  </property>
  <widget class="QPushButton" name="btn_analyze">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始训练模型和分析</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_log">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_import">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_file_path">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>请选择文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
